### ISSUE TITLE
Issues with GetFiles() throwing " DirectoryNotFoundException" on Android and stopping at the topmost folder

### ISSUE THREAD

**Issue by leoncvlt** [gwiazdorrr/BetterStreamingAssets#17]
I am calling GetFiles() to get a list of all my StreamingAssets files and folders - to do so I copied the xample code from the repository readme and changed it for my use case:
`
string[] paths = BetterStreamingAssets.GetFiles("\\", "*", SearchOption.AllDirectories); ` - it worked just fine on the editor, but on Android I got this exception:

```
DirectoryNotFoundException: Attempted to access a path that is not on the disk. 
   at BetterStreamingAssets+ApkImpl.GetFiles (System.String path, System.String searchPattern, System.IO.SearchOption searchOption) [0x00190] in <697df9379d4040eeb82e91f9e9d87c04>:0
   at BetterStreamingAssets.GetFiles (System.String path, System.String searchPattern, System.IO.SearchOption searchOption) [0x00000] in <697df9379d4040eeb82e91f9e9d87c04>:0
```

I found that it would work if I changed the "\\" path argument to a foward slash "/" it seemed to fix it, but then I noticed that it would only return the files in the topmost folder (in this case, the StreamingAssets folder root.

Here's my StreamingAssets folder contents:
```
│   assays.json      
│   assays.json.meta 
│   graphics.meta    
│   test.json        
│   test.json.meta   
│                    
└───graphics         
        0001.png     
        0001.png.meta
        0002.png     
        0002.png.meta
        0003.png     
        0003.png.meta
        0004.png     
        0004.png.meta
        0005.png     
        0005.png.meta
        0006.png     
        0006.png.meta
        0007.png     
        0007.png.meta
        0008.png     
        0008.png.meta
```

Only the 2 .json files are returned -  If I do copy those .png files to the root folder then they are returned as well. 

Moreover, if I try and call `GetFiles()` on the graphics folder, it throws `DirectoryNotFoundException` no matter what I put in the path - I tried "graphics", "/graphics", "\\graphics", "graphics\" and so on - nothing really worked.

Using Unity 2019.2.18f, and a device on Android 8.

**Comment by leoncvlt** [comment]
Alright, I ran a few more tests with this - I noticed that the graphics folder wasn't even read by the `GetStreamingAssetsInfoFromJar()` method when going through the jar assets. Here's what I tried:

- I created another folder called "hello" with a bunch of text files inside it and that one was being found.
- I renamed the "graphics" folder to something else and now it's being detected.
- Thought that "graphics" was a special folder name (like "bin"?) so figured that was the issue. But then I renamed the folder back to "graphics" and it was still working!

Now I can't seem to break it anymore. I guess something with renaming the folder fixed it (maybe .meta files related?).

So that's it for my little adventure. Sorry if I wasted your time. I guess the issue with the forward slash as path still remains though 👀
